### ISSUE TITLE
hello-libs fails to build with Studio 3.4 Canary 6

### ISSUE THREAD

**Issue by arashpayan** [googlesamples/android-ndk#597]
OS: Ubuntu 18.04
Android Studio: 3.4 Canary 6

I had trouble with my own project after updating to canary 6 this afternoon. My project is very much like hello-libs, so I tried building it with canary 6 to see if the problem was unique to my project, and it also failed. The error is:

```
Build command failed.
Error while executing process /home/arash/apps/android-sdk/cmake/3.6.4111459/bin/cmake with arguments {--build /home/arash/coding/zood/android-ndk/hello-libs/app/.externalNativeBuild/cmake/debug/x86 --target hello-libs}

ninja: error: '/home/arash/coding/zood/android-ndk/hello-libs/app/distribution/gmath/lib/x86/libgmath.a', needed by '/home/arash/coding/zood/android-ndk/hello-libs/app/build/intermediates/cmake/debug/obj/x86/libhello-libs.so', missing and no known rule to make it
```

Repro steps:
1. Clone the android-ndk repo
2. Using Android Studio 3.4, Canary 6 open up the hello-libs project
3. When Studio asks you to update Gradle, say yes.
4. Build the project

Expected: The project should build
Actual: The build fails with the error shown above

Unfortunately, I don't know enough about CMake or Gradle to figure out if there is problem there or if it's a bug in Studio. If I can provide any more information, please don't hesitate to ask.

**Comment by ggfan** [comment]
thank you for raising issue and, most of all, trying out canary! will have a look

**Comment by arashpayan** [comment]
Thanks @ggfan. I starred the new issue you filed.

Using `CMAKE_CURRENT_SOURCE_DIR` does allow me to get back to work in the meantime. Thanks for the tip. :smile:

**Comment by ggfan** [comment]
merged the changes into samples:  studio 3.4 implementation seems moving forward as is.  Will need to create documentation on Developer.Android.Com to reflect it at the change.
